### PR TITLE
Jb49299

### PR DESCRIPTION
**Pull request by martyone**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -4,15 +4,9 @@ ARG SDK_VERSION
...
-RUN env
-RUN ssh-keygen -A
-RUN usermod -u $LOCALUID mersdk
-RUN rm /lib/systemd/system/sysinit.target.wants/*
-RUN (cd /lib/systemd/system/multi-user.target.wants/; for i in *; do [ $i == sshd-keys.service ] || [ $i == sshd.socket ] || rm -f $i; done);
-RUN rm /etc/systemd/system/basic.target.wants/*
-RUN (cd /etc/systemd/system/multi-user.target.wants/; for i in *; do [ $i == sdk-webapp.service ] || rm -f $i; done);
-RUN (cd /lib/systemd/system/sockets.target.wants/; for i in *; do [ $i == dbus.socket ] || rm -f $i; done);
-RUN (cd /lib/systemd/system/basic.target.wants/; for i in *; do [ $i == dbus.service ] || rm -f $i; done);
+RUN env; \
```
Comment (vigejolla) [inline Dockerfile]: The env command was just for debugging purposes, I think it could be safely removed now.

File: setup-buildengine.sh [hunk 2]
```diff
@@ -185,8 +187,18 @@ createTar() {
     fi
     mkdir -p mer.d
     sudo mount /dev/nbd0p1 mer.d
+
+    echo "Disabling unneeded systemd services ..."
+    sudo rm mer.d/lib/systemd/system/sysinit.target.wants/*
+    sudo rm mer.d/lib/systemd/system/multi-user.target.wants/!(sshd-keys.service|sshd.socket)
```
Comment (vigejolla) [inline setup-buildengine.sh]: nice :)

**Comment by martyone** [comment]
> Commit [124eb3c](https://github.com/sailfishos/sdk-build-tools/commit/124eb3c94802c10cbaf9f57880ba25182b6ca1fd) belongs to some other PR?

No, though related to some other bug, here it is just a refactorization step to enable applying the functional changes cleanly.

File: Dockerfile (hunk 1 above)
Comment (martyone) [reply Dockerfile]: Dropped

**Pull request merged**